### PR TITLE
Clean up code style in formatters

### PR DESCRIPTION
**Pull request by adidahiya**

### CODE REVIEW

File: src/formatters/proseFormatter.ts [hunk 1]
```diff
@@ -14,22 +14,21 @@
  * limitations under the License.
 */
 
-/// <reference path='../../lib/tslint.d.ts' />
+///<reference path='../../lib/tslint.d.ts' />
```
Comment (ashwinr) [inline src/formatters/proseFormatter.ts]: this is incorrect, you're missing a space after the three slashes

File: src/formatters/verboseFormatter.ts [hunk 2]
```diff
@@ -14,23 +14,22 @@
  * limitations under the License.
 */
 
-/// <reference path='../../lib/tslint.d.ts' />
+///<reference path='../../lib/tslint.d.ts' />
```
Comment (ashwinr) [inline src/formatters/verboseFormatter.ts]: space

**Comment by adidahiya** [comment]
@ashwinr fixed

File: src/formatters/jsonFormatter.ts [hunk 3]
```diff
@@ -14,15 +14,11 @@
  * limitations under the License.
 */
 
-/// <reference path='../../lib/tslint.d.ts' />
+///<reference path='../../lib/tslint.d.ts' />
```
Comment (ashwinr) [inline src/formatters/jsonFormatter.ts]: sigh, fix this here and below.

**Comment by adidahiya** [comment]
sigh, sorry about all that

**Comment by adidahiya** [comment]
@ashwinr ?

**Pull request merged**